frontend/components/icons/Apple.tsx

```diff
@@ -1,10 +1,33 @@
 import React from "react";
 
-const Apple = () => {
+interface IAppleProps {
+  size?: "small" | "medium" | "large";
+  color?: "coreFleetBlack" | "coreVibrantBlue" | "uiFleetBlack75";
+}
+
+const FLEET_COLORS = {
+  coreFleetBlack: "#192147",
+  coreVibrantBlue: "#6a67fe",
+  uiFleetBlack75: "#515774",
+};
+
+const WIDTH_SIZE_MAP = {
+  small: "12",
+  medium: "16",
+  large: "24",
+};
+
+const HEIGHT_SIZE_MAP = {
+  small: "12.5",
+  medium: "17",
+  large: "25",
+};
```

Comment:
Should the width and height be the same?

Reply:
Good call! When I built this it was 16x17 on figma, but I see these icons are now on the official style guide so I'll update them and PM you!